vsts/pipelines/templates/_releaseStepTemplate.yml

```diff
@@ -58,6 +58,13 @@ steps:
     scriptPath: ./vsts/scripts/tagBuildpacksImagesForRelease.sh
   condition: and(succeeded(), eq(variables['ReleaseBuildImages'], 'true'))
 
+- task: ShellScript@2
+  displayName: 'Test runtime images'
+  inputs:
+    scriptPath: ./build/testRunTimeImages.sh
```

Comment:
nit:
I know it's still WIP, but giving a heads up just in case :) 
instead of running all the runtime tests again (which will increase the ci time) won't it be better to just run the version tests in our runtime images?

e.g. NodeImage_Contains_VersionAndCommit_Information, PythonRuntimeImage_Contains_VersionAndCommit_Information
 and VersionMatchesImageName (in php)

Reply:
I'll take a look at adding an additional attribute on the subset of tests we want to run for the release step and using that as a part of the filter when calling `dotnet test`.

Reply:
Added a trait to the tests that we want to run during the release step in [this commit](https://github.com/microsoft/Oryx/pull/385/commits/fdbdc62a35f44ff1bd27beecf7d3c1a594e54dd8).

Reply:
The subset of runtime tests we're running during the release step only adds 90 seconds to the build (compared to the ~27 minutes the full set would take 🎉)